samples/cpp/benchmark_app/statistics_report.hpp

```diff
@@ -118,4 +150,20 @@ class StatisticsReport {
 
     // csv separator
     std::string _separator;
+
+    StatisticsReport::PerformanceCounters get_average_performance_counters(
+        const std::vector<PerformanceCounters>& perfCounts);
+};
+
+class StatisticsReportJSON : public StatisticsReport {
+public:
+    explicit StatisticsReportJSON(Config config) : StatisticsReport(std::move(config)) {}
+
+    void dump() override;
+    void dump_performance_counters(const std::vector<PerformanceCounters>& perfCounts) override;
+
+private:
+    void dump_parameters(nlohmann::json& js, const StatisticsReport::Parameters& parameters);
+    const nlohmann::json StatisticsReportJSON::perf_counters_to_json(
+        const StatisticsReport::PerformanceCounters& perfCounts);
```

Comment:
```suggestion
    const nlohmann::json perf_counters_to_json(
        const StatisticsReport::PerformanceCounters& perfCounts);
```
this fixes next error:
```
/home/ivikhrev/Work/openvino/samples/cpp/benchmark_app/statistics_report.hpp:167:26: error: extra qualification ‘StatisticsReportJSON::’ on member ‘perf_counters_to_json’ [-fpermissive]
     const nlohmann::json StatisticsReportJSON::perf_counters_to_json(
```